src/GovLst.sol

```diff
@@ -681,6 +690,10 @@ abstract contract GovLst is IERC20, IERC20Metadata, IERC20Permit, Ownable, Multi
   /// 2. Caller's claimAndDistributeReward transaction is in the mempool.
   /// 3. The payoutAmount is changed.
   /// 4. The claimAndDistributeReward transaction is now included in a block.
+  ///
+  /// The number of deposits can grow beyond being processed in a single transaction. This can cause the total payout of
```

Comment:
Might suggest some slight rewording:

"The number of deposits owned by the LST may grow to such a size that all deposits cannot be included in a single call. This can cause..."